GliaWidgets/Sources/ViewController/Common/Alert/AlertViewController.swift

```diff
@@ -27,10 +27,28 @@ class AlertViewController: UIViewController {
             accepted: () -> Void,
             declined: () -> Void
         )
+
+        /// Indicating whether an alert is as a result on user action,
+        /// like pressing "End" engagement button, or pressing stop screen sharing button.
+        /// Based on that user-initiated can be only the `confirmation`case
+        fileprivate var isUserInitiated: Bool {
```

Comment:
Maybe this property can be `private (set)`? This may allow unit testing of it as well.

Reply:
This is enum property, so it can be stored